test/test_rpc.py

```diff
@@ -0,0 +1,170 @@
+import sys
+import tempfile
+
+import torch
+import torch.distributed as dist
+import torch.multiprocessing as mp
+
+from common_utils import TestCase, load_tests, run_tests
+from common_utils import NO_MULTIPROCESSING_SPAWN
+
+# load_tests from common_utils is used to automatically filter tests for
+# sharding on sandcastle. This line silences flake warnings
+load_tests = load_tests
+
+if not dist.is_available():
+    print('c10d not available, skipping tests')
+    sys.exit(0)
+
+
+if NO_MULTIPROCESSING_SPAWN:
+    print('spawn not available, skipping tests')
+    sys.exit(0)
+
+class RpcTest(TestCase):
+
+    world_size = 2
+
+    @classmethod
```

Comment:
nit: You know there is a `@staticmethod` which doesn't give you a `cls` argument. You can use that if you don't actually need to know the `cls`.